### DISCUSSION TITLE
Where on Amazon.com do you opt-in for delivery update e-mails?

### DISCUSSION THREAD

**Discussion by GizmoGoody** [moralmunky/Home-Assistant-Mail-And-Packages#662] [Q&A] [answered]
On my Amazon account page, I can see the "Shipment updates via text" and have clicked through all of the other options like "Email subscriptions" and "Communication preferences," but I never find where I can opt-in for delivery update e-mails. Where is it?

![image](https://user-images.githubusercontent.com/46235745/166074402-b34baacb-694d-4f86-90a3-c3dd7e6201fe.png)

**Comment by firstof9** [comment, marked as answer]
I thought they were automatic?

**Comment by GizmoGoody** [comment]
@firstof9 I think I've got it working again. I'll know the next time I place an order. Thanks.

@moralmunky I see the Amazon forwarded email addresses field is for "the email addresses that {I} will forward amazon notification from..." If I have Amazon set to e-mail delivery updates directly to the same e-mail address I set in Step 1 (i.e. no forwarding required), should I just repeat that same address in this field? I've tried to leave it blank, but the configuration screen won't allow me to submit.

**Comment by firstof9** [comment]
Enter ""

**Reply by GizmoGoody** [reply]
Thanks

Edit: Two quotations "" didn't work but two single quotes '' did.
